5_databases/week2/homework_2_part1.sql

```diff
@@ -0,0 +1,27 @@
+-- Active: 1706440038860@@127.0.0.1@3307@hyf_lesson1
+-- Add a task with these attributes: title, description, created, updated, due_date, status_id, user_id
+-- Change the title of a task
+-- Change a task due date
+-- Change a task status
+-- Mark a task as complete
+-- Delete a task
+
+INSERT INTO
```

Comment:
you are missing an ERD diagram. so the comment you placed on the diff does not show the ERD. you can just add the ERD file to the repo like any other file, commit, and push.